ecosystem/ffmpeg_plugin/mtl_st20p_rx.c

```diff
@@ -148,6 +164,21 @@ static int mtl_st20p_read_header(AVFormatContext* ctx) {
   dbg(ctx, "%s, fb_cnt: %d\n", __func__, s->fb_cnt);
   ops_rx.framebuff_cnt = s->fb_cnt;
 
+#ifdef MTL_GPU_DIRECT_ENABLED
+  if (s->gpu_direct_enabled) {
+    // create contex for one gpu device
```

Comment:
```suggestion
    /* create context for one gpu device */
```

Reply:
Yup i saw that too 
but there are comments used in both styles 

I also wonder about unifying the header guard comment style 
but i wonder if i should do this in this comment ? 

atomic commits say that this is not a part of it